### ISSUE TITLE
Unpin segyio==1.9.3

### ISSUE THREAD

**Issue by jondequinor** [equinor/semeio#274]
segyio was pinned due to packaging issues. If these are fixed, unpin. If not, an upgrade require building segyio from source.

**Comment by lars-petter-hauge** [comment]
From my personal mac:
```
 ~/venvs  python3 -m venv test_seg_env
 ~/venvs  source test_seg_env/bin/activate
(test_seg_env)  ~/venvs  python --version
Python 3.8.6
(test_seg_env)  ~/venvs  pip install --upgrade pip
Collecting pip
  Using cached pip-20.3.1-py2.py3-none-any.whl (1.5 MB)
Installing collected packages: pip
  Attempting uninstall: pip
    Found existing installation: pip 20.2.1
    Uninstalling pip-20.2.1:
      Successfully uninstalled pip-20.2.1
Successfully installed pip-20.3.1
(test_seg_env)  ~/venvs  pip install segyio
Collecting segyio
  Downloading segyio-1.9.5-cp38-cp38-macosx_10_9_x86_64.whl (85 kB)
     |████████████████████████████████| 85 kB 1.6 MB/s
Collecting numpy>=1.10
  Using cached numpy-1.19.4-cp38-cp38-macosx_10_9_x86_64.whl (15.3 MB)
Installing collected packages: numpy, segyio
Successfully installed numpy-1.19.4 segyio-1.9.5
```

Perhaps we were a bit quick on the pinning of segyio here? could it be pip that needed an upgrade instead? (in case we forgot to do that..)

**Comment by oyvindeide** [comment]
> Perhaps we were a bit quick on the pinning of segyio here? could it be pip that needed an upgrade instead? (in case we forgot to do that..)

It was pinned because the wheels were missing for osx, that problem has been fixed now.

**Comment by oyvindeide** [comment]
Closed in #275
